obspy/io/ascii/core.py

```diff
@@ -44,7 +44,7 @@
 from obspy.core.util import AttribDict, loadtxt
 
 
-HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %d sps, %.26s, %s, %s, %s\n"
+HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %s sps, %.26s, %s, %s, %s\n"
 
```

Comment:
In [7]: "%s" % (12.0000000001)
Out[7]: '12.0000000001'

In [8]: "%s" % (12.00000000001)
Out[8]: '12.0'